rwgame/debugstate.cpp

```diff
@@ -285,6 +290,34 @@ Menu* DebugState::createAIMenu()
 	return m;
 }
 
+Menu*DebugState::createWeaponMenu()
+{
+	CharacterObject* player = nullptr;
+	if (game->getPlayer()) {
```

Comment:
if this condition is false player will be `nullptr` in the lambda function.
I think you mean to do this check in the lambda function below.